packages/app/src/app/store/sequences.js

```diff
@@ -187,19 +182,19 @@ export const forkSandbox = sequence('forkSandbox', [
   },
 ]);
 
+function stopFrozenSandboxFromEdit() {
```

Comment:
@SaraVieira This can be an arrow function

```suggestion
const stopFrozenSandboxFromEdit = () => {
```